js/chart.js

```diff
@@ -18,7 +20,9 @@ export const drawChart = async (rooms, date = null) => {
   );
   const close_time = moment.max(
     rooms.map(room => {
-      const hours = room.location.hours[date.day()].endhours;
+      const hours = (
+        room.location.hours.find((el) => el.day === date.day())
```

Comment:
Can you leave a comment on this change? What does it fix?

Reply:
Before the way it was pulling dates would mean the hours would possible get set to the hours for another day. 

For example if the location was closed on Sunday (day 0) so the `location.hours` did not contain 7 entries but 6. This instead uses the `hours.day` to find the correct matching day.

This also fixes the issue with the Ballentine location crashing the draw chart because the array would be out of bounds on Sunday / Saturday when you try to view those days by setting a default hour range to 9 - 21h.